systemvm/debian/opt/cloud/bin/setup/router.sh

```diff
@@ -61,6 +61,10 @@ setup_router() {
     then
       log_it "Reloading udev for new udev NIC assignment"
       udevadm control --reload-rules && udevadm trigger
+      if [ "$HYPERVISOR" == "vmware" ]; then
+          sync
+          reboot
```

Comment:
Since we are in systemd time now I would use:

<pre>systemctl reboot</pre>

But hey, that's a very, very, very minor thing.